GVFS/GVFS.Platform.Mac/MacPlatform.cs

```diff
@@ -91,16 +91,14 @@ public override NamedPipeServerStream CreatePipeByName(string pipeName)
 
         public override IEnumerable<EventListener> CreateTelemetryListeners(string providerName, string enlistmentId, string mountId)
         {
-            // TODO: return TelemetryDaemonEventListener when the telemetry daemon has been implemented for Mac
-
-            // string gitBinRoot = this.GitInstallation.GetInstalledGitBinPath();
-            // var daemonListener = TelemetryDaemonEventListener.CreateIfEnabled(gitBinRoot, providerName, enlistmentId, mountId, pipeName: "vfs");
-            // if (daemonListener != null)
-            // {
-            //     yield return daemonListener;
-            // }
-
-            yield break;
+            string gitBinRoot = this.GitInstallation.GetInstalledGitBinPath();
+
+            const string pipeName = "/usr/local/GitService/pipe/vfs-c780ac06-135a-4e9e-ab6c-d41e2d265baa";

```

Comment:
This should be read from config and not hardcoded in the product, similar to how the `GVFSTelemetryId` is managed.

Reply:
Moved it to a new config setting `gvfs.telemetry-pipe` as not to interfere with the existing `telemetry-id` setting used by the ETW listener.

If `telemetry-pipe` is set to not-null then we'll take that as the pipe name and try sending to it, otherwise telemetry via pipe is disabled.